README.md

```diff
@@ -1,132 +1,184 @@
+    return self;
+}
+```
 
-  5. ```- (NSArray *)removeFriends: (NSArray *)friends;```
-    * Remove the friends listed in the passed in argument from our current `friends` property.
-    * Return an array of friends who were found and removed
+**Top-tip:** *The* `NSMutableArray` *ivar should be initialized to an empty* `NSMutableArray` *using* `[[NSMutableArray alloc] init];
 
-#### Playing with your methods in the AppDelegate
+3 — Write the implementation for the convenience initializer `initWithName:age:`. Remember that this method should:
 
-Feel free to play with your new class outside of the tests in the AppDelegate. Here is a reminder how.
+  * assign to `self` a call of the designated initializer upon `self`, passing in the arguments for `name` and `age` directly into the designated initializer, and passing in default values for `skills` and `qualifiedTutor` into the designated initializer, and
+  * return `self`.
 
-  1. Import the Person class in the AppDelegate's .m 
-  2. In didFinishLaunchingWithOptions, instantiate a new Person. ie. ```Person *al = [[Person alloc] init];```  
-  3. Set the person's age, height, name, and isFemale properties using dot notation.  ie. ```al.height = @72;``` 
-  4. Test each of the required methods and NSLog the output.  ie. ```NSLog(@"%@",[al grow]) // ex output 76;```
-  5. Create a few additional Person objects to test addFriends ie. ```[al addFriends:@[jon, chris, sally]]; //assuming i already created jon, chris and sally.  
+```objc
+// convenience initializer
+-(instancetype)initWith... {
+    self = [self initWith...]; // calls designated initializer
+    return self;
+}
+```
 
-## Hints 
+### III. Implement the `celebrateBirthday` Method
 
-Here is a helper method for random floats 
+1 — Copy the following (private) helper method `ordinalForInteger:` into the `.m` implementation file immediately below the empty implementation for `celebrateBirthday`. This method returns a string containing the correct ordinal for the `NSUInteger` submitted as the `integer` argument:
 
 ```objc
-#define ARC4RANDOM_MAX 0x100000000
-- (CGFloat)randomFloatBetweenNumber:(CGFloat)minRange andNumber:(CGFloat)maxRange
-{
-    return ((float)arc4random() / ARC4RANDOM_MAX)
-    * (maxRange - minRange)
-    + minRange;
+// helper method
+- (NSString *)ordinalForInteger:(NSUInteger)integer {
+    NSString *ordinal = @"th";
+    if (integer % 10 == 1 && integer % 100 != 11) {
+        ordinal = @"st";
+    } else if (integer % 10 == 2 && integer % 100 != 12) {
+        ordinal = @"nd";
+    } else if (integer % 10 == 3 && integer % 100 != 13) {
+        ordinal = @"rd";
+    }
+    return ordinal;
 }
 ```
 
-Check out the documentation on NSMutableArray (Help -> Documentation and API Refrences).  NSMutableArray has some great methods ie. addObject, addObjectsFromArray, removeObject, removeObjectsFromArray 
+2 — Write the method implementation for `celebrateBirthday` to:
```

Comment:
This is slightly weird one...  I like it for fun value, but the naming/return stuff is weird. @jmburges opinions?